README.md

```diff
@@ -0,0 +1,116 @@
+# Cilium OLM
+
+> NOTE: this documentation is for Cilium maintainers, the user guide for OpenShift is part of [Cilium documentation][okd-gsg]
+
+This repository contains Cilium packaging for OpenShift, which is centred around [Operator Lifecycle Management APIs (OLM)][olm].
+
+## Key Components
+
+On OpenShift Cilium has to be installed by an installer operator in order to comply with RedHat certification requirements.
+
+To install Cilium on OpenShift user has to have the following:
+
+- standard Kubernetes manifests for Cilium installer operator - RBAC, Deployment, Service ...etc
+- OpenShift-specific manifests for the installer operator - Subscription, OperatorGroup, ClusterServiceVersion
+
+### Installer operator 
+
+The installer operator is based on RedHat helm-operator, which was chose for expediance. In the future it's expected that Cilium will have a general-purpose installation operator that will support OpenShift as well as other platforms.
+
+The operator is configured using a [`CiliumConfig` CRD](config/crd/cilium.io_cilumconfigs.yaml). The operator watches a single `CiliumConfig` CR in `cilium` namespace. The CRD is a simple object that hold Helm values.
+
+Default CR specs for each release are sourced from top-level manifest (e.g. [`ciliumconfig.v1.10.yaml`](ciliumconfig.v1.10.yaml)), a copy is made for each release in `manifests/` (e.g. [`manifests/cilium.v1.9.6/cluster-network-07-cilium-ciliumconfig.yaml`](manifests/cilium.v1.9.6/cluster-network-07-cilium-ciliumconfig.yaml).
+
+### Cofiguration source code 
+
+Since configuration for OLM was fairly complex to manage in static YAML format, it was written in [CUE](http://cuelang.org/).
+CUE is not very easy to use on its own, so a simple opinionated utility called [`kg`][kg] is being used to execute CUE templates and write out YAML files.
+
+The CUE templates can be found in [`config/operator/`]. `kg` is invoked with [a helper script](scripts/generate-configs.sh), which is typically driven by `make`, e.g. `make generate.configs.v1.9.6`.
+
+### Generated configuration
+
+Configuration that is generated from CUE templates will get used in variety of different ways:
+
+- contents of `mnanifests/` dir are used during installation in user guides as well as CI
+- contents of `bundles/` dir are used for building metadata bundle images
+
+### Images
+
+The installation is reliant on operator application image, it is currently based on RedHat's helm-operator.
+
+For the purposes of certfication, the operator application image is accompanied by a metadata bundle image. Both images have to undergo certification tests in RedHat Partner Connect registry, maintainers should be able obtain access to this registry.
```

Comment:
Links to the RedHat Partner Connect registry would be good

Reply:
I've added a section with links.